src/main/java/it/gov/pagopa/pu/bff/mapper/OrganizationDTOMapper.java

```diff
@@ -10,11 +11,23 @@
 public class OrganizationDTOMapper {
 
   public OrganizationDTO mapToOrganizationDTO(Organization organization, List<String> roles) {
+
+    String operatorRoleValue = (roles != null && !roles.isEmpty()) ? roles.get(0) : null;
+
+    OrganizationDTO.OperatorRoleEnum operatorRole = null;
+    if (operatorRoleValue != null) {
+      try {
+        operatorRole = OrganizationDTO.OperatorRoleEnum.fromValue(operatorRoleValue);
+      } catch (IllegalArgumentException e) {
+        throw new InvalidOperatorRoleException("INVALID_OPERATOR_ROLE: " + operatorRoleValue);
+      }
+    }
```

Comment:
move into a separate method
before to take the first role, let's search if it contains ROLE_ADMIN and return it, if it doesn't, take the first